src/main/js/controls/PairedResult/helpers/helpers.js

```diff
@@ -547,16 +539,36 @@ const clickHandler = (graphContext, control, config, canvasSVG) => (
             shownTargets.push(item.key);
         }
     };
-    legendClickHandler(element);
+    const legend = legendClickHandler(element);
     updateShownTarget(config.shownTargets, item);
-    canvasSVG
-        .selectAll(`path[aria-describedby="${item.key}"]`)
-        .attr("aria-hidden", true);
-    canvasSVG
-        .selectAll(`.${styles.pairedPoint}[aria-describedby="${item.key}"]`)
-        .attr("aria-hidden", true);
+    const value = item.key.substr(0, item.key.lastIndexOf("_"));
```

Comment:
Is there a better way to find out whether an item is high/median/mid/low instead of having to rely on the string manipulation here? Is there a property or something that can be accessed? 

cc / @abhijit945

Reply:
I am sorry why do you need `value`. AFAIK you are just getting the `key` arent you? 
I am pretty sure I dont have any data ids associated with only `key` for Paired Results.

Reply:
`value` which I am changing to the `parentKey` is needed, as the key that I am receiving is either of these `[uid_1_high, uid_1_mid, uid_1_low, uid_2_high, uid_2_mid, uid_2_low]`. But the line which joins the high and low point of paired result is created with either `[uid_1, uid_2]`. So to get either of these `[uid_1, uid_2]` from the `item`, I am using string manipulation. Because when `mid` point is removed or added into the graph, the line shouldn't be affected.

![Screenshot 2020-01-10 at 11 32 38 AM (2)](https://user-images.githubusercontent.com/29273628/72129625-f960b200-339c-11ea-8407-b0b7f52bc681.png)

Reply:
I see, then couldn't we do the same thing as `shouldCreateLine`?

Reply:
Updated here https://github.com/cerner/carbon-graphs/pull/140/commits/aee60a4616d37bef8e43d816734beebf93c2da59.